Symfony/CS/Tests/BasicFileFixingTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests;
+
+use Symfony\CS\Fixer;
+
+/**
+ * Test the behavior of all fixers on some basic PHP files.
```

Comment:
Maybe it no longer has a use, but given the bugs found and fixed in this PR and this one; https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1978 the test had a purpose.
I'll delete it